lib/presentation/crowdaction/crowdaction_comments/parts/comment_actions.dart

```diff
@@ -0,0 +1,68 @@
+part of '../crowdaction_comments_page.dart';
+
+/// A list of all actions that can be displayed about or done on a comment.
+///
+/// These include:-
+///  - The comment date can be Today or the yyyy-mm-dd formatted date
+///  of the comment. (TODO: The date formatting isn't yet confirmed from design)
+///  - The number of likes, this displays the like count only when it's 1 or more.
+///  - Reply option, this requests focus to the comment text field, keeping a
+///  record of which comment is being responded to.
+///  - Flag option [CollactionIcons.flag] when not flagged and
+///  [CollactionIcons.flag_filled] when the issue is flagged.
```

Comment:
About this, we should consider comments only in a flagged state when flagged by the user viewing it. Just a note, the backend isn't ready for this quite yet anyhow.

Reply:
Yea, for now we'll just use place holders.